### PR TITLE
golang mysql examples [T633]

### PR DESCRIPTION
**Pull request by Lagovas**

### CODE REVIEW

File: examples/golang/src/example_with_zone/example_with_zone.go [hunk 1]
```diff
@@ -18,103 +18,128 @@ package main
...
-	}
-	var zone, data []byte
-	var rawData string
-	fmt.Println("zone, data - raw_data")
-	for rows.Next() {
-		err := rows.Scan(&zone, &data, &rawData)
+	if *data != "" {
+		rand.Seed(time.Now().UnixNano())
+		//get new zone over http
+		resp, err := http.Get("http://127.0.0.1:9191/getNewZone")
```
Comment (vixentael) [inline examples/golang/src/example_with_zone/example_with_zone.go]: is it the address of acraconnector? shall we put it into variable?

**Comment by vixentael** [comment]
What is the proper way of running this example?
Acra should be on and running on localhost machine? As well as database?

File: examples/golang/src/example_with_zone/example_with_zone.go (hunk 1 above)
Comment (Lagovas) [reply examples/golang/src/example_with_zone/example_with_zone.go]: yes. it's acraconnector's address and it used once, so I didn't assign it to variable and used default value

**Comment by Lagovas** [comment]
yes. this is simple example that was created when Acra could work only with AcraConnector in the pair and also required db) can be used any our docker config where used AcraConnector + AcraServer + Database

**Comment by Lagovas** [comment]
> What is the proper way of running this example?
> Acra should be on and running on localhost machine? As well as database?

actually, I remembered that we explain python examples in our wiki - https://github.com/cossacklabs/acra/wiki/Using-Acra-to-Protect-Your-Django-App#if-you-want-to-try-acra-in-containers-first
And all examples are the same. They differ only by running program (python/ruby/golang). Other are the same actually.

**Pull request merged**